src/components/Pivot/Pivot.tsx

```diff
@@ -0,0 +1,93 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import '../../yamui';
+import * as React from 'react';
+import { NestableBaseComponentProps } from '../../util/BaseComponent/props';
+import { Pivot as FabricPivot, PivotItem as FabricPivotItem } from 'office-ui-fabric-react/lib/Pivot';
+
+import './Pivot.css';
+
+export interface PivotItem {
+  ariaLabel?: string;
+  key: string;
+  text: string;
+}
+
+export interface PivotProps extends NestableBaseComponentProps {
+  /**
+   * Indicates whether the text on the tabs will be rendered on white colors
+   * to be used with dark backgrounds.
+   */
+  isDark?: boolean;
+  /**
+   * Indicates the key of the initially selected tab or pivot item.
+   */
+  selectedKey: string;
+  /**
+   * Contains the information of the pivot items that will be rendered within this component.
+   */
+  pivotItems: PivotItem[];
+  /**
+   * Callback that will be executed every time there is a change in the selected pivot item.
+   */
+  onChange(key: string): void;
+}
+
+export interface PivotState {
+  /**
+   * Key of the selected pivot item at a certain point.
+   */
+  selectedKey: string;
+}
+
+export default class Pivot extends React.Component<PivotProps, PivotState> {
+  constructor(props: PivotProps) {
+    super(props);
+    this.onLinkClick = this.onLinkClick.bind(this);
+
+    this.state = {
+      selectedKey: this.props.selectedKey,
```

Comment:
there's a similar conversation happening in rich's ChoiceGroup PR. you shouldn't need to manage any state here, you can just let the app rendering this component pass selectedKey. the app can manage state via onChange and pass the new selectedKey value down

Reply:
Got it, I added this prop to state to keep track of the latest selected key, and only fire the 'onChange' event when the key changed, but I think I can do the comparison with props.selectedKey, right?

Reply:
yep, that should work fine

Reply:
Alrighty, I've addressed the comments